src/codeu/chat/server/Server.java

```diff
@@ -112,6 +116,15 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
 
         final String name = Serializers.STRING.read(in);
         final User user = controller.newUser(name);
+        
+        String userAddCommand = "U-ADD ";
+        userAddCommand += user.id + " ";
+        userAddCommand += user.name + " ";
+        userAddCommand += user.creation;
+

```

Comment:
I'm not sure if Time is also a necessary component to include? (I was wondering this for my c-add as well!) @wrdana